tst/EnergyPlus/unit/WaterThermalTanks.unit.cc

```
@@ -732,7 +732,7 @@ TEST_F(EnergyPlusFixture, HPWHEnergyBalance)
     state->init_state(*state);
     state->dataEnvrn->StdRhoAir = 1.0;
 
-    ASSERT_FALSE(WaterThermalTanks::GetWaterThermalTankInput(*state));
+    WaterThermalTanks::GetWaterThermalTankInput(*state);
```

No need to wrap into "EXPECT_NO_THROW". We just rely on gtest catching the exception if needed. This is what we do in most tests right now

---

Yep, let it fail if it's gonna fail.